docs/cni/kubernetes/GCE.md

```diff
@@ -0,0 +1,156 @@
+
+## 1. Getting started with GCE
+These instructions describe how to set up two CoreOS hosts on GCE.  For more general background, see 
+[the CoreOS on GCE documentation][coreos-gce].
+
+### 1.1 Install the gcloud tool
+If you already have the `gcloud` utility installed, and a GCE project configured, you may skip this step.
+
+Download and install GCE, then restart your terminal: 
+```
+curl https://sdk.cloud.google.com | bash
+```
+For more information, see Google's [gcloud install instructions][gcloud-instructions].
+
+Log into your account:
+```
+gcloud auth login
+```
+
+In the GCE web console, create a project and enable the Compute Engine API.  
+Set the project as the default for gcloud:
+```
+gcloud config set project PROJECT_ID
+```
+And set a default zone
+```
+gcloud config set compute/zone us-central1-a
+```
+
+### 1.2 Setting up GCE networking
+GCE blocks traffic between hosts by default; run the following command to allow Calico traffic to flow between 
+containers on different hosts:
+```
+gcloud compute firewall-rules create calico-ipip --allow 4 --network "default" --source-ranges "10.240.0.0/16"
+```
+You can verify the rule with this command:
+```
+gcloud compute firewall-rules list
+```
+
+<!--- master only -->
+### 1.3 Clone this project
+
+    git clone https://github.com/projectcalico/calico-containers.git
+<!--- else
+### 1.3 Clone this project, and checkout the **release** release
+
+    git clone https://github.com/projectcalico/calico-containers.git
+    git checkout tags/**release**
+<!--- end of master only -->
+    
+## 2. Deploy the VMs 
+Change into the directory for this guide.
+```
+cd calico-containers/docs/cni/kubernetes/
+```
+
+Deploy the Kubernetes master node using the following command:
+```
+gcloud compute instances create \
+  kubernetes-master \
+  --image-project coreos-cloud \
+  --image coreos-stable-835-9-0-v20151208 \
+  --machine-type n1-standard-1 \
+  --metadata-from-file user-data=cloud-config/master-config.yaml
+```
+
+Deploy at least one worker node using the following command:
+```
+gcloud compute instances create \
+  kubernetes-node-1 \
+  --image-project coreos-cloud \
+  --image coreos-stable-835-9-0-v20151208 \
+  --machine-type n1-standard-1 \
+  --metadata-from-file user-data=cloud-config/node-config.yaml
+```
+
+## 3. Using your cluster
+### 3.1 Configuring kubectl
+The following steps configure remote kubectl access to your cluster.
+
+Download `kubectl`
+```
+wget https://storage.googleapis.com/kubernetes-release/release/v1.1.4/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+```
+
+The following command sets up SSH forwarding of port 8080 to your master node so that you can run `kubectl` commands on your local machine.
+```
+gcloud compute ssh kubernetes-master --ssh-flag="-nNT" --ssh-flag="-L 8080:localhost:8080" &
```

Comment:
Hm.  According to the `gcloud` docs, running `gcloud compute ssh` should do all the magic in terms of generating an ssh key for you.  Do you mean it was asking for a passphrase for the new key it was going to generate?

Looks like we can guard against this by using the `--quiet` option which won't ask for a passphrase.